elements/lisk-transactions/src/register_delegate.ts

```diff
@@ -54,7 +53,9 @@ const validateInputs = ({
 		);
 	}
 
-	validateNetworkIdentifier(networkIdentifier);
+	if (hexToBuffer(networkIdentifier).length !== 32) {
+		throw new Error('Invalid network identifier length');
```

Comment:
Maybe add the expected length to the message so it is more informative.

Reply:
I think having valid length won't help much to the user in this case.
Also, this function and in commander, it needs to be updated when we design for the UI.

I will keep the current code for this PR, but let's discuss the validation when we rewrite for the UI